test/Mono.Linker.Tests.Cases.Expectations/Assertions/KeptOverrideAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Assertions
+{
+	[AttributeUsage (AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
+	/// <Summary>
+	/// Used to ensure that a method should keep an 'override' annotation for a method in the supplied base type
+	/// Fails in tests if the method doesn't have the override method in the original or linked assembly
```

Comment:
Please add a comment that this behaves differently from the other `Kept` attributes - almost all the other ones test both sides, that is if the attribute is not there it means the thing should NOT be in the output. For example if a certain type doesn't have a `Kept` attribute on it, we expected the type to not survive linking.
We do have some other attributes which are like this, but I think it's a good idea to at least document it - and maybe mention the other attribute RemoveOverrideAttribute as its counterpart.